src/tests/integration/test-data/foundry/test/FreshInt.t.sol

```diff
@@ -30,4 +33,32 @@ contract FreshIntTest is Test, KEVMCheats {
         int128 val = int128(uint128(kevm.freshUInt(16)));
         assertGt(val, max);
     }
+
+    function manip_symbolic_bytes(bytes memory b) public {
+        uint middle = b.length / 2;
+        b[middle] = hex'aa';
+    }
+
+    function test_symbolic_bytes_function() public {
+        bytes memory fresh_bytes = kevm.freshBytes(5);
+        manip_symbolic_bytes(fresh_bytes);
+        assertEq(hex'aa', fresh_bytes[2]);
+    }
+
+    function test_symbolic_bytes_1() public {
+        bytes memory fresh_bytes_1 = kevm.freshBytes(5);
+        assertEq(fresh_bytes_1.length, 5);
+        local_byte = fresh_bytes_1[0];
+
+        bytes memory fresh_bytes_2 = kevm.freshBytes(67);
+        assertEq(fresh_bytes_2.length, 67);
+        local_bytes = fresh_bytes_2;
+    }
+
+    function test_symbolic_bytes_length(uint256 l) public {
+        vm.assume(l <= 64);
+        vm.assume(l > 0);
```

Comment:
Why these limits?

Reply:
Generating a 0 length array doesn't work, and there is an arbitrary small upper limit so the proof doesn't run too long, since this test generates a byte-copying loop. I presume it would be possible to write a circularity lemma for this, but I fear that would add to the fragility of the test.

Reply:
Very well, but let's put something that's not a multiple of 32 so that more complexity is exercised, or have one test with a multiple of 32 and one with a non-multiple.

Reply:
I think this is important particularly because we don't have an invariant in place.

Reply:
Changed the test to maximum length 48.